www-client/seamonkey/Manifest

```diff
@@ -1,3 +1,10 @@
+AUX all-gentoo-1.js 1008 BLAKE2B 50418db37f55b9b9e53049abb95af79a5ee5137df9aab3ef6d1791e51a0c2f35d78b390982c5c20f4a57acebfaa013139276d0343cd4733a925510c8a5567c69 SHA512 5d1c9331db3c988f79be7756b57254bb3d2cb56cf10c7e79878d9968f882713cdcca0013d3507c4d69fcc7e99164a27b05ffbbb5ce78ee86b91ecce366a997c4
```

Comment:
For your local overlay, enable 'thin manifests' to avoid this junk in here (AUX/MISC/etc), it should only be DIST entries.